dev/com.ibm.ws.kernel.feature_fat/publish/servers/com.ibm.ws.kernel.feature.fat.versionless.minify/server.xml

```diff
@@ -0,0 +1,10 @@
+<server>
+    <include location="../fatTestPorts.xml"/>
+    
+    <featureManager>
+        <platform>jakartaee-10.0</platform>
+        <feature>servlet</feature>
+        <feature>ejb</feature>
```

Comment:
```suggestion
        <feature>ejbLite</feature>
```
Switch to ejbLite since enterpriseBeanLite-4.0 is already in the tested.features in bnd.bnd